src/com/hms_networks/americas/sc/taginfo/TagInfoEnumeratedIntToString.java

```diff
@@ -0,0 +1,80 @@
+package com.hms_networks.americas.sc.taginfo;
+
+import java.util.ArrayList;
+
+public class TagInfoEnumeratedIntToString extends TagInfo {
+
+  /** Integer values can be used to represent string values in a string mapping */
+  private final String[] enumeratedStringValueMapping;
+
+  /**
+   * Constructor for tag class with group booleans and with enumerated string value mappings.
+   *
+   * @param id tag ID
+   * @param name tag name
+   * @param historicalLogEnabled boolean if historical logging enabled
+   * @param isInGroupA boolean if tag in group A
+   * @param isInGroupB boolean if tag in group B
+   * @param isInGroupC boolean if tag in group C
+   * @param isInGroupD boolean if tag in group D
+   * @param type tag type
+   * @param enumeratedStringValueMapping array of strings where the integer value of the tag
+   *     represents the string array index
+   */
+  public TagInfoEnumeratedIntToString(
+      int id,
+      String name,
+      boolean historicalLogEnabled,
+      boolean realTimeLogEnabled,
+      boolean isInGroupA,
+      boolean isInGroupB,
+      boolean isInGroupC,
+      boolean isInGroupD,
+      TagType type,
+      String[] enumeratedStringValueMapping) {
+

```

Comment:
remove extra line before `super(`

Reply:
Extra (and inconsistent) blank line still present: https://github.com/hms-networks/sc-flexy-tag-info-lib/blob/e47446ef7f3077f91debc313da9f45140fd2aa83/src/com/hms_networks/americas/sc/taginfo/TagInfoEnumeratedIntToString.java#L42